package-manifest-spec.md

```diff
@@ -0,0 +1,156 @@
+3. Package manifests will primarily be used by package indexes to popululate basic package metadata.
+2. Package Managers will typically be used to assist in the creation and modification of Package Manifest documents.
+4. Package Manifest documents will occasionally be created and modified manually by people.
+4. Package manifests will typically be stored alongised the package source code.
+
+
+## Format
+
+The canoonical format for the package manifest is a JSON document containing a
+single JSON object.  
+
+
+## Filename
+
+Convention is to name this document `epm.json` which is short for *'ethereum
+package manifest'*.
+
+
+## Document Specification
+
+The following fields are defined for the package manifest.  Custom fields may
+be included.  Custom fields **should** be prefixed with `x-` to prevent name
+collisions with future versions of the specification.
+
+
+### Manifest Version: `manifest_version`
+
+
+The `manifest_version` field defines the version of the ethereum package manifest
+specification (this document) that this manifest conforms to. All manifests
+**must** include this field.
+
+* Key: `manifest_version`
+* Type: Integer
+* Allowed Values: `1`
+* Package Manager Guide: Package managers should validate this field is valid prior to publishing new manifests.
+
+
+### Package Name: `package_name`
+
+The `package_name` field defines a human readable name for this package.  All
+manifests **should** include this field.
+
+* TODO: what should happen if this is missing (should it be required?)
+* Key: `package_name`
+* Type: String
+
+### Authors: `authors`
+
+The `authors` field defines a list of human readable names for the authors of
+this package.  All manifests **should** include this field. 
+
+
+* Key: `authors`
+* Type: List of Strings
+
+
+### Version: `version`
+
+The `version` field declares the current version number of this package.  This value
+**should** be conform to the [semver](http://semver.org/) version numbering
+specification.  All manifests must **include** this field.
+
+* Key: `version`
+* Type: String
+
+
+### Description: `description`
+
+The `description` field *should* be used to provide additional detail that may be relevant for the package.
+
+* TODO: should this include a suggestion to use a specific markup format like markdown?
+* Key: `description`
+* Type: String
+
+
+### Description: `keywords`
+
+The `keywords` field *should* be used to provide relevant keywords related to this package.
+
+* Key: `keywords`
+* Type: List of Strings
+
+
+### Links: `links`
+
+TODO: This could potentially not be necessary, or should be removed.  Needs discussion.
+
+The `links` field *should* be used to provide URIs to relevant resources
+associated with this package.  When possible, authors *should* use the
+following keys for the following common resources.
+
+* `website`: Primary website for the package.
+* `documentation`: Package Documentation
```

Comment:
@VoR0220 this would be something like `https://web3py.readthedocs.io/en/latest/`.  A URI to a website or other resource that represents the formal documentation for the package.